src/packages/data-type/workspace/data-type-workspace.context.ts

```diff
@@ -301,9 +328,15 @@ export class UmbDataTypeWorkspaceContext
 		this.#currentData.update({ name });
 	}
 
+	getPropertyEditorSchemaAlias() {
+		return this.#currentData.getValue()?.editorAlias;
+	}
 	setPropertyEditorSchemaAlias(alias?: string) {
 		this.#currentData.update({ editorAlias: alias });
 	}
+	getPropertyEditorUiAlias() {
+		return this.#currentData.getValue()?.editorUiAlias;
+	}
```

Comment:
@nielslyngsoe would it be simpler to havd a single function `getPropertyEditor()` which retun both properties?

So if both it needed one only has to call a single function.

```
getPropertyEditorUiAlias() {
const currentData = this.#currentData.getValue()?.editorUiAlias;

if (!currentData) return null
return { currentData.editorAlias, currentData.editorUiAlias }
	}
```

Reply:
Hi @bjarnef 

I guess you can say that, but coming from a general architectural point of view then we have many 'get...' methods of our different contexts. And if we had to make these different combinations then we would end up with a lot of code.
I would say if there was a very specific, highly used scenario where we needed both, then it would make sense. But currently I'm not sure even sure we use this method. So I do not thing we need this kind of optimization currently.